### ISSUE TITLE
single_build.groovy conan client call

### ISSUE THREAD

**Issue by keelerad** [lasote/skynet_example#8]
Thanks for updating the example, looks like I had correctly guessed the correct new syntax to use (as I have got my svn+ssh checkout of source based example working in Jenkins) but was wondering about the -s directives you have put on the conan create call

`client.run(command: "create . lasote/stable -s compiler.version=\"9.1\" -s compiler=\"apple-clang\" -s compiler.libcxx=\"libc++\" -pr \"" + conf_repo_dir + "/" + params.profile + "\"")`

I had similar issues with it complaining about various compiler undefineds but I sorted by adding them to the profile files, is defining a target specific set in the example a good idea ?

Also I passed through user/channel from multi build by adding it as a parameter to the call to single_build 

```
            	string user_channel = ref.split("@")[1]
                sshagent (['release-ssh-key']){
                	client.run(command: "create . " + user_channel + " --update --profile \"" + conf_repo_dir + "/" + params.profile + "\"")
                }
```

I'll remove the --update from mine

Alec Keeler, Build and Release Engineer Nexor Ltd

**Comment by lasote** [comment]
You are right, I've removed the "-s" parameters, sorry probably they were used in some debugging task.
